src/lib/turtle.js

```diff
@@ -1053,6 +1055,21 @@ function generateTurtleModule(_target) {
         };
         proto.$shape.minArgs     = 0;
         proto.$shape.co_varnames = ["name"];
+        //colormode supported
+        proto.$colormode = function(cmode){
+            if(cmode !== undefined){
+                if(cmode === 255) {
+                    this._colorMode = 255;
+                } else {
+                    this._colorMode = 1.0;
+                }   
+                return this.addUpdate(undefined, this._shown, {colorMode : this._colorMode});         
+            }
+
+            return this._colorMode===255?Sk.builtin.int_(255):Sk.builtin.float_(1.0);
+        }
+        proto.$colormode.minArgs     = 0;
+        proto.$colormode.co_varnames = ["mode"];
```

Comment:
It is preferred to use the `returnType` decorator to supply information on how values should be cast to Sk types as opposed to returning Sk types directly.

```js
proto.$colormode.returnType = function(value) {
    return value === 255 ? Sk.builtin.int_(255) : Sk.builtin.float_(1.0);
};
```
This keeps the turtle implementation as "pure" javascript so we don't have to do remember to do any type juggling if we call other methods from within this file.